### DISCUSSION TITLE
Automatically signed up to AI Minds newsletter?

### DISCUSSION THREAD

**Discussion by MeowcaTheoRange** [deepgram/community#330] [General help] [answered]
### Which Deepgram product are you using?

Other

### Details

**How did you get my email?** I don't remember signing up to this newsletter, as I only sign up to newsletters I find relevant to my interests. AI is most definitely not one of my interests.
I don't even have a Deepgram account.
![Screenshot_20230829-112555-01](https://github.com/deepgram/community/assets/58280776/924a18c4-ec1c-4e90-8d14-c8b0138a0271)

**Comment by jpvajda** [comment, marked as answer]
@MeowcaTheoRange  Hello! AI Minds is a newsletter from Deepgram that is sent to anyone that has opted in to receiving updates from the company. If you didn't sign up for receiving updates from Deepgram we apologize for the inconvenience. 

If you'd like to unsubscribe, you can do so using the unsubscribe link provided in the newsletter email and you will be removed from the distribution list.
